packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -933,6 +939,8 @@
 					"${PODS_CONFIGURATION_BUILD_DIR}/React-graphics/React_graphics.framework/Headers",
 				);
 				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
+				LD = "";
+				LDPLUSPLUS = "";
```

Comment:
This change should be reverted.

Reply:
[8c7547e](https://github.com/facebook/react-native/pull/45114/commits/8c7547ec8823356c1578e79384ec159b1e10c817)